### DISCUSSION TITLE
TFT 35 E3 V3.0 Y-axis "Move" and "Home" travel in opposite directions

### DISCUSSION THREAD

**Discussion by Wolfang-Max** [bigtreetech/BIGTREETECH-TouchScreenFirmware#2160] [Q&A]
Hi, this is my first post and I hope someone can help me. 
I have installed a SKR Mini E3 V2.0 and a TFT 35 E3 V3.0 on an older i3 style printer. 
My X and Z axes are moving in the expected direction when using the arrow keys of the TFT display, and they also move to the "0" position upon a Home command from the TFT. But my Y axis will either move opposite to the arrow on the TFT display, with the Home command from the TFT moving correctly towards the "0" position (switch), or, when I change the plug on the Y motor, the Y axis will move in the right direction according to the arrows, but will move away from the Y end-switch when Homing from the TFT.
How can I correct this? I have tried to edit the config,h file but have not been successful.
Thank you,
Wolfgang
P.S. please don't laugh at me if the solution is ease. I am a pretty old guy and these things become harder to understand.

**Comment by kisslorand** [comment]
There is no Back and Forward button but Up and Down.  Up button should move the bed forward and Down button should move it backward. Is this your case? If yes than everything is fine, this is how it should be.

**Reply by Wolfang-Max** [reply]
Thanks kisslorand,
you are correct. Pressing the Up button moves the bed towards the back, and pressing the Down button moves it towards the front.
The problem is that if I press the Y axis "Home" button, the bed will move away from the microswitch end stop, rather than towards it, as both the other axes do until they hit their respective end stop switches and then come to a stop.
My end stops are in the most common positions, left for X, back for Y and a capacitive probe for Z.
Where in the Marlin do I tell the machine in which direction to move when "Homing"?
Thanks,
Wolfgang

**Comment by kisslorand** [comment]
You should invert Y axis direction by hardware changes (plug, pins, etc.)

**Comment by dokcal** [comment]
I guess this is by design--and certainly, since zero is at the back, the Y value is going "up" as the bed moves forward--but it's very counterintuitive. I wish there was some way to alter this behavior in firmware!

**Comment by radek8** [comment]
@Wolfang-Max 
It might help if you interchanged the names on the Y-axis icons

**Reply by dokcal** [reply]
That is freakin' BRILLIANT! Good thinking!

**Comment by kisslorand** [comment]
I might make a PR to change the Y axis icons. It is counterintuitive indeed. I always mix them, as my TFT is tilted the UP icon also somehow indicates back and the DOWN arrow faces a little forward.
